docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -0,0 +1,332 @@
+== Introduction to devfile
+
+What is a devfile?
+
+A https://redhat-developer.github.io/devfile/[devfile] is a portable file that describes your initial development environment. It allows for a _portable_ developmental environment without the need of reconfiguration.
```

Comment:
do we want to use the word `initial`? Because that might imply that devfiles do not get updated through out the process of development and behave like a starter kit?